src/browser/theme.js

```diff
@@ -49,7 +49,8 @@ define((require, exports, module) => {
     'daringfireball.net': ['#fff', '#4a525a', IS_DARK],
     'firewatchgame.com': ['#EF4338', '#2D102B', IS_DARK],
     'whatliesbelow.com': ['#fff', '#74888B', IS_DARK],
-    'supertimeforce.com': ['#2EBCEC', '#051224', IS_DARK]
+    'supertimeforce.com': ['#2EBCEC', '#051224', IS_DARK],
+    'huffingtonpost.com': ['#fff', '#2E7360', IS_DARK]
```

Comment:
I don't think we want to give this a dark background. The goal is to blend in with the page and draw the eye away from controls.